dev-lang/fbc/fbc-1.06.0.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="FreeBASIC - A free/open source, multi-platform BASIC compiler."
+HOMEPAGE="http://www.freebasic.net"
+SRC_URI="https://github.com/freebasic/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/freebasic/${PN}/releases/download/${PV}/FreeBASIC-${PV}-source-bootstrap.tar.xz"
+
+LICENSE="FDL-1.2 GPL-2+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
You'll probably need to request arm and arm64 keywords for a new package, unless it's already approved by arm teams.